WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -443,10 +445,8 @@ - (NSString *)sortDateFromPostDictionary:(NSDictionary *)dict
  */
 - (NSString *)featuredImageFromPostDictionary:(NSDictionary *)dict
 {
-    NSString *featuredImage = @"";
+    NSString *featuredImage = [NSString string];
```

Comment:
I know `@""` over `[NSString string]` is just a personal thing, since they're both equivalent. But i really like this small change!